plugins/@grouparoo/mailchimp/src/initializers/plugin.ts

```diff
@@ -10,13 +10,13 @@ import emailDestination from "../lib/export/connection";
 import idDestination from "../lib/export-id/connection";
 import importSource from "../lib/import/connection";
 
+const templateRoot = path.join(__dirname, "..", "..", "public", "templates");
```

Comment:
To keep things tidy we might want to move variable definitions after all the imports